crates/cairo-profiler/src/sierra_loader.rs

```diff
@@ -33,6 +35,17 @@ impl SierraProgram {
     }
 }
 
+#[allow(dead_code)]
+#[derive(Default, Clone)]
+pub struct StatementsFunctionsMap(HashMap<StatementIdx, Vec<String>>);
```

Comment:
What is `Vec<String>`?

Reply:
Will change to `FunctionName`, good idea